<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>986</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WFEM2TSn</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rec.qrc">
    <normaloff>:/new/prefix1/image/logo.ico</normaloff>:/new/prefix1/image/logo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <row>
           <property name="text">
            <string>1</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>2</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>3</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>4</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>5</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>6</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>7</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>8</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>9</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>10</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>11</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>12</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>13</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>字节编码</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>名称</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>数据类型</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>数值</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>备注</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>0~7</string>
           </property>
          </item>
          <item row="0" column="1">
           <property name="text">
            <string>时间</string>
           </property>
          </item>
          <item row="0" column="2">
           <property name="text">
            <string>8个Uint8</string>
           </property>
          </item>
          <item row="0" column="4">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="1" column="0">
           <property name="text">
            <string>8~9</string>
           </property>
          </item>
          <item row="1" column="1">
           <property name="text">
            <string>设备序列号</string>
           </property>
          </item>
          <item row="1" column="2">
           <property name="text">
            <string>Uint16</string>
           </property>
          </item>
          <item row="2" column="0">
           <property name="text">
            <string>10~11</string>
           </property>
          </item>
          <item row="2" column="1">
           <property name="text">
            <string>时间样点数</string>
           </property>
          </item>
          <item row="2" column="2">
           <property name="text">
            <string>Uint16</string>
           </property>
          </item>
          <item row="2" column="4">
           <property name="text">
            <string>单个记录单个通道采样个数</string>
           </property>
          </item>
          <item row="3" column="0">
           <property name="text">
            <string>12</string>
           </property>
          </item>
          <item row="3" column="1">
           <property name="text">
            <string>通道数</string>
           </property>
          </item>
          <item row="3" column="2">
           <property name="text">
            <string>Uint8</string>
           </property>
          </item>
          <item row="3" column="3">
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item row="3" column="4">
           <property name="text">
            <string>Ch1(WE)、Ch2(NS)</string>
           </property>
          </item>
          <item row="4" column="0">
           <property name="text">
            <string>13</string>
           </property>
          </item>
          <item row="4" column="1">
           <property name="text">
            <string>记录头长度</string>
           </property>
          </item>
          <item row="4" column="2">
           <property name="text">
            <string>Uint8</string>
           </property>
          </item>
          <item row="4" column="3">
           <property name="text">
            <string>32</string>
           </property>
          </item>
          <item row="4" column="4">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="5" column="0">
           <property name="text">
            <string>14</string>
           </property>
          </item>
          <item row="5" column="1">
           <property name="text">
            <string>记录状态指示</string>
           </property>
          </item>
          <item row="5" column="2">
           <property name="text">
            <string>Uint8</string>
           </property>
          </item>
          <item row="5" column="3">
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item row="6" column="0">
           <property name="text">
            <string>15~16</string>
           </property>
          </item>
          <item row="6" column="1">
           <property name="text">
            <string>饱和位标识</string>
           </property>
          </item>
          <item row="6" column="2">
           <property name="text">
            <string>Uint16</string>
           </property>
          </item>
          <item row="6" column="3">
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item row="6" column="4">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="7" column="0">
           <property name="text">
            <string>17</string>
           </property>
          </item>
          <item row="7" column="1">
           <property name="text">
            <string>采样字节数</string>
           </property>
          </item>
          <item row="7" column="2">
           <property name="text">
            <string>Uint8</string>
           </property>
          </item>
          <item row="7" column="3">
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item row="7" column="4">
           <property name="text">
            <string>3个字节</string>
           </property>
          </item>
          <item row="8" column="0">
           <property name="text">
            <string>18~19</string>
           </property>
          </item>
          <item row="8" column="1">
           <property name="text">
            <string>采样率</string>
           </property>
          </item>
          <item row="8" column="2">
           <property name="text">
            <string>Uint16</string>
           </property>
          </item>
          <item row="8" column="4">
           <property name="text">
            <string>单位：Hz</string>
           </property>
          </item>
          <item row="9" column="0">
           <property name="text">
            <string>20</string>
           </property>
          </item>
          <item row="9" column="1">
           <property name="text">
            <string>采样率单位</string>
           </property>
          </item>
          <item row="9" column="2">
           <property name="text">
            <string>Uint8</string>
           </property>
          </item>
          <item row="9" column="3">
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item row="9" column="4">
           <property name="text">
            <string>0：秒</string>
           </property>
          </item>
          <item row="10" column="0">
           <property name="text">
            <string>21</string>
           </property>
          </item>
          <item row="10" column="1">
           <property name="text">
            <string>时钟状态</string>
           </property>
          </item>
          <item row="10" column="2">
           <property name="text">
            <string>Uint8</string>
           </property>
          </item>
          <item row="10" column="3">
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item row="10" column="4">
           <property name="text">
            <string/>
           </property>
          </item>
          <item row="11" column="0">
           <property name="text">
            <string>22~25</string>
           </property>
          </item>
          <item row="11" column="1">
           <property name="text">
            <string>时钟误差</string>
           </property>
          </item>
          <item row="11" column="2">
           <property name="text">
            <string>Int32</string>
           </property>
          </item>
          <item row="11" column="3">
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item row="11" column="4">
           <property name="text">
            <string>有符号数的，单位为ms</string>
           </property>
          </item>
          <item row="12" column="0">
           <property name="text">
            <string>26~31</string>
           </property>
          </item>
          <item row="12" column="1">
           <property name="text">
            <string>保留字</string>
           </property>
          </item>
          <item row="12" column="2">
           <property name="text">
            <string>Char*6</string>
           </property>
          </item>
          <item row="12" column="3">
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item row="12" column="4">
           <property name="text">
            <string>必须为0</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="7">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Band:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="6">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBoxRecord">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>3600</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>记录块个数：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5" colspan="2">
           <widget class="QComboBox" name="comboBoxBand"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>时间片长度：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinBoxSlot">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>3600</number>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QComboBox" name="comboBoxAMTorMT">
            <item>
             <property name="text">
              <string>AMT</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>MT</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>AMT/MT:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenFile"/>
   <addaction name="actionConvert"/>
   <addaction name="actionConsult"/>
  </widget>
  <action name="actionOpenFile">
   <property name="text">
    <string>打开文件</string>
   </property>
   <property name="toolTip">
    <string>打开广域时间域文件</string>
   </property>
  </action>
  <action name="actionConvert">
   <property name="text">
    <string>转换</string>
   </property>
  </action>
  <action name="actionConsult">
   <property name="text">
    <string>查看TSn</string>
   </property>
   <property name="toolTip">
    <string>前往TSn文件目录</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="rec.qrc"/>
 </resources>
 <connections/>
</ui>
